src/app/registration/facility-select/facility-select.component.ts

```diff
@@ -112,20 +118,25 @@ export class FacilitySelectComponent implements OnInit {
   get bookingOpeningHour(): number {
     // As a temporary work-around, date rules for each park are currently based on
     // the first open facility at the park. See BRS-570 for details
+    // Use formatted booking time for warning when out booking window. 
     const facility = this.facilities?.find(f => f.status.state === 'open');
     let bookingOpeningHour = this.defaultAMOpeningHour;
-
     if (facility && (facility.bookingOpeningHour || facility.bookingOpeningHour === 0)) {
       bookingOpeningHour = facility.bookingOpeningHour;
     }
-
+    const dateTime = DateTime.fromObject(
+      { hour: this.defaultAMOpeningHour, minute: 0 },
+      { zone: 'America/Vancouver' } // PST zone
+    );
+    this.formattedBookingTime = dateTime.toFormat("h:mm a 'PDT'");
     return bookingOpeningHour;
   }
 
-  get isOpeningHourPast(): boolean {
-    // check the current time in the America/Vancouver TZ (must do this step to acct for PST/PDT)
-    const currentHour = this.getPSTDateTime().get('hour');
-    return Boolean(parseInt(currentHour, 10) >= this.bookingOpeningHour);
+  get isOpeningHourPast(): boolean { 
+    if (this.systemTimePST != null){
```

Comment:
NB: Spacing

`if (this.systemTimePST != null){` => `if (this.systemTimePST != null) {`

Reply:
Resolved in [Commit 43beab5](https://github.com/bcgov/parks-reso-public/pull/461/commits/43beab52a1c7d818910d777c2133aefa47ce60ff)